.gitignore

```diff
@@ -15,4 +15,5 @@ target/
 .project
 .classpath
 metastore_db/
-derby.log
\ No newline at end of file
+derby.log
+integration/spark-common-test/src/test/resources/Data
```

Comment:
Remove this file..it is not required to be committed

Reply:
Are there any bad effect for carbon? Please tell me if have

When I run SDV in local machine, I need copy DATA directory to carbondata project, and except DATA directory when I git add and commit. So I add this line to .gitignore. I suggest add this line if not bad effect for carbondata project.  

Or there are any better method to solve this problem, please tell me too.

Reply:
I removed this line